packages/oui-collapsible/src/collapsible.controller.js

```diff
@@ -42,7 +43,22 @@ export default class {
             .on("resize", () => this.$scope.$apply());
     }
 
+
+    close () {
+        if (this.expanded) {
```

Comment:
I feel like if this method is called and the component is not open, then we should send an error, instead of ignoring the programming error: https://en.wikipedia.org/wiki/Fail-fast

Reply:
This method is mainly there for the accordion mode and don't need to send error. 
All we want, is to have each item of the same group in the `closed` state before opening a new one. If there is one or more opened, we toggle it (and send the event callback). We don't want to get an error each time it's called if the item is already closed.

In this case, we just call it for each component of the same group without checking before if it's opened or not.